api/lib/infrastructure/repositories/organization-learner-repository.js

```diff
@@ -357,11 +357,38 @@ const findAllLearnerWithAtLeastOneParticipationByOrganizationId = async function
   return new ParticipantRepartition(result);
 };
 
+const findAllLearnerWithAtLeastOneParticipationByOrganizationIds = async function (organizationIds) {
```

Comment:
**thought** : on pourrait à terme merger ces 2 méthodes en une qui prend un tableau d'organizationId (à faire plus tard)

Reply:
-> On a noté le refacto à faire par la suite si les perfs sont ok !